self-hosting/index.md

```diff
@@ -163,3 +163,11 @@ RESET_PASSWORD_URL=https://licensegate.example.com/auth/password
 CORS_ORIGIN=https://licensegate.example.com
 GOOGLE_AUTH_CLIENT_ID=none
 ```
+
+## Setting up your plugin with the java wrapper
+LicenseGate, by default, uses its own license server that is provided under `api.licensegate.io`. In order to use your own licensing server, you'll need to setup the follwing:
+```java
+LicenseGate licenseGate = new LicenseGate("YOUR_USER_ID");
+licenseGate.setValidationServer("https://license.yourdomain.com");
+```
+Once this is done, license validation should now go through your own licensing server instead of the cloud-hosted one from LicenseGate.
```

Comment:
I'd prefer not to mention "plugins" in the official docs to keep in general. 

Just a general remark might be sufficient:

## Usage with wrappers
LicenseGate wrapper libraries, by default, use the public LicenseGate server which is provided under `api.licensegate.io`. In order to use your self-hosted licensing server, you will need to configure your server url when using wrappers.
Refer to the wrapper documentation for detailed instructions.

Reply:
Done, additionally changed the "setValidationServer" example to `backend.example.com` to further specify it to be the backend.

I have also expanded the RSA-Key example to include further information on not to strip the beginning and end-sections of the key.

Reply:
Done